src/DataLayouts/copyto.jl

```diff
@@ -61,31 +65,39 @@ function Base.copyto!(
         Base.Broadcast.Broadcasted{Style}(bc.f, bc.args, ()),
     )
     @inbounds bc0 = bc[]
-    fill!(dest, bc0)
-    call_post_op_callback() && post_op_callback(dest, dest, bc, to)
+    fill!(dest, bc0, mask)
+    call_post_op_callback() && post_op_callback(dest, dest, bc, to, mask)
 end
 
 #####
 ##### DataLayouts
 #####
 
+compute(::NoMask, idx) = true
+compute(mask, idx) = !iszero(mask[idx])
```

Comment:
This is minor, but to me the name `compute` implies that some computation is being done, rather than just checking if computation should be done. Maybe something like `mask_check_compute` would be more informative

Reply:
Yeah, I punted on the name because I couldn't think of a good one. A name that I think is descriptive would be `skip_compute`, but I prefer and try to avoid double negatives (e.g., `!skip_compute`)

Reply:
Or `is_active`?

Reply:
I like `should_compute`!

Reply:
Updated!